### ISSUE TITLE
Issues with input devices

### ISSUE THREAD

**Issue by Norkybork** [MiSTer-devel/AtariST_MiSTer#13]
Current version of core won't allow me to use any input device (keyboard/mouse/pad) the issue is intermittent. 

Other cores work flawlessly

After some testing with previous versions of the core anything after 202110109 doesn’t work for me.  

For me this is by far the best core for the mister. Keep up the good work

**Comment by sorgelig** [comment]
cannot confirm. Works here both keyboard and mouse with latest core, main and linux

**Comment by Norkybork** [comment]
Yes others have also said everything is working for them on the latest core.  But for me it doesn't and as far as I'm aware I have all the latest updates through update all.  My system is pretty much stock.  

As long as I have a working system I guess.

**Comment by sorgelig** [comment]
From Linux system which provides input event to core it doesn't matter which core is running. If Keyboard and Mouse work in other cores, then it should work in AtariST too.. 
I have no idea what's wrong.. May be some specific settings/bios in AtariST
